controllers/apply/awards-for-all/fields.js

```diff
@@ -630,6 +630,52 @@ module.exports = function fieldsFor({ locale, data = {} }) {
         };
     }
 
+    function fieldContactLanguagePreference(props) {
```

Comment:
Might be useful to extract this out into `fields/`, aiming to break things out gradually so would be good to do for new fields.

Reply:
Good call, give it a look now (plus tests).